Mathlib/Topology/CompletelyRegular.lean

```diff
@@ -95,3 +95,10 @@ instance T4Space.instT35Space [T4Space X] : T35Space X := by
   have : T1Space X := T2Space.t1Space
   have : CompletelyRegularSpace X := NormalSpace.instCompletelyRegularSpace
   exact {}
+
+lemma separatesPoints_continuous_of_T35Space [T35Space X] :
+    SeparatesPoints (Continuous : Set (X → ℝ)) := by
+  intro x y x_ne_y
+  obtain ⟨f, f_cont, f_zero, f_one⟩ :=
+    CompletelyRegularSpace.completely_regular x {y} isClosed_singleton x_ne_y
+  refine ⟨fun x ↦ f x, continuous_subtype_val.comp f_cont, by aesop⟩
```

Comment:
Should this be spelled `exact` instead of `refine` or is there a benefit to using `refine` here?

Reply:
`exact` can be used here, and is preferred over `refine` to _close_ a proof.
```suggestion
  exact ⟨fun x ↦ f x, continuous_subtype_val.comp f_cont, by aesop⟩
```

Reply:
Thanks! (And sorry about that sloppiness in such a small PR.)